src/main/java/org/reactome/release/qa/common/SkipList.java

```diff
@@ -0,0 +1,78 @@
+package org.reactome.release.qa.common;
+
+import org.gk.model.GKInstance;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkipList {
+
+    public List<String> skipList = new ArrayList<>();
+
+    /**
+     * The SkipList class contains functions for reading in an assumed skiplist, and checking its contents.
+     * The main assumption is that the skiplist file will be stored in a "resources" folder, and
+     * be in the format "step_name_skip_list.txt".
+     * @param displayName - String, displayName of class calling SkipList class.
+     * @throws IOException, thrown when file does not exist. Note: Does not terminate program.
+     */
+    public SkipList(String displayName) throws IOException {
+        if (Files.exists(getSkipListFilePath(displayName))) {
+            skipList = readDbIdsFromSkipListFile(getSkipListFilePath(displayName));
+        } else {
+            FileNotFoundException e = new FileNotFoundException();
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * This method gets file name for skiplist based on two assumptions. The first assumption is that it will
+     * be a file located in the "resources" directory. The second is that the skip list, if it exists, will
```

Comment:
Where does "resources" need to be, relative to the executable? In the same directory, or in a subdirectory?

Reply:
The `resources` folder needs to be in the same directory as the executable. This is in line with other 'configuration file'-based methods: https://github.com/reactome/release-qa/blob/master/src/main/java/org/reactome/release/qa/common/AbstractQACheck.java#L40

Reply:
So it needs to be in the same directory as the JAR. I'd clarify that in the comment.

Reply:
I misspoke, its not that it needs to be in same directory as the executable, but in the same directory as the _executor_. I amended the comment in the following way:

```The main assumption is that the skiplist file will be stored in a "resources" folder, which needs to be located in the same directory as the executor, and be in the format "step_name_skip_list.txt".```